packages/base-contract/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { abiUtils, BigNumber } from '@0x/utils';
```

Comment:
Were we able to remove abiUtils and ethers.js entirely from our codebase?

Reply:
`abiUtils` is still used to rename overloaded methods and assert correct encoding.

`ethers` is used by `deployAsync`.
There is a task to remove this from `deployAsync` after which we can remove ethers.js.